data/korath/korath missions.txt

```diff
@@ -257,3 +257,555 @@ mission "Small Scale Delivery"
 	on complete
 		payment 1000
 		dialog `You find a Korath who seems to have been waiting for you in the spaceport. They greet you by showing their palms, then outstretch their hands as if waiting to receive the package. You hand off the small delivery and they hand you a small payment of <payment>.`
+
+mission "First Contact: Korath Exiles"
+	landing
+	source "Far'en Lai"
```

Comment:
```suggestion
	source "Far'en Lai"
	to offer
		not "Wanderers: Sestor: Exiles 2: active"
```
Probably shouldn't have this pop up when you've got Rek with you.

Reply:
I think MZ and @ravenshining wrote mission was before Wanderers middle. There are quite a few problems with it.

Reply:
That mission is a part of Wanderers Middle, since January 2017. It's a pretty integral part to that string, making contact with the Exiles to get help with the Kor Sestor. It may get reworked/cleared from the game for any Wanderers changes, but for now, it's been a part of the game for over 6 years.

Reply:
Yes, I'm just saying it isn't the only ancient problem in the mission.

Reply:
I have merged your suggestion. I don't know why I'm still seeing the "Commit suggestion" button, but I can assure you that your fix is in there.